access-control-app/src/cmr/access_control/services/acl_util.clj

```diff
@@ -14,6 +14,7 @@
     [cmr.common-app.services.search.query-model :as qm]
     [cmr.transmit.echo.tokens :as tokens]
     [cmr.transmit.metadata-db2 :as mdb]
+    [cmr.transmit.metadata-db :as mdb1]
```

Comment:
Are we planning on keeping these two namespaces around indefinitely? @ygliuvt - do you know the plan?